src/app/components/Searchfield/Searchfield.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classes from './Searchfield.module.css';
+
+type InputProps = {
+  placeholder: string;
+};
+
+const imageSrc = (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="15.644"
+    height="16.204"
+    viewBox="0 0 15.644 16.204"
+  >
+    <path
+      id="Search_icon"
+      data-name="Search icon"
+      d="M16.39,14.759l-3.856-4.011a6.54,6.54,0,1,0-5.007,2.336A6.472,6.472,0,0,0,11.274,11.9l3.886,4.041a.853.853,0,1,0,1.23-1.183ZM7.526,1.707A4.836,4.836,0,1,1,2.691,6.542,4.841,4.841,0,0,1,7.526,1.707Z"
+      transform="translate(-0.984)"
+      fill="#fff"
+    />
+  </svg>
+);
+
+function Searchfield({ placeholder }: InputProps): JSX.Element {
+  return (
+    <label className={classes.search}>
+      <div className={classes.search__image}>{imageSrc}</div>
```

Comment:
I would expect that `imageSrc` is something we use in `<img src={imageSrc} />`, but it's not a string.
It is a JSX.Element. I would prefer to give it a good name like `SearchIcon` and outsource it.

```suggestion
      <div className={classes.search__image}><SearchIcon /></div>
```

If you are able to remove the `div`, it would be even better:

```suggestion
      <SearchIcon className={classes.search__image} />
```

Reply:
We changed the variables name but we didnt outsource it on purpose! ;-)